package.json

```
@@ -20,6 +20,7 @@
     "generate:backend:notification-models": "shx rm -rf generated/notifications && shx mkdir -p generated/notifications && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.16.0/api_notifications.yaml --out-dir generated/notifications",
     "generate:backend:notification-hub": "shx rm -rf generated/notifications && shx mkdir -p generated/notifications && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.16.0/notification_queue_messages.yaml --out-dir generated/notifications",
     "generate:templates:mailvalidation": "TARGET=generated/templates/mailvalidation/index && VERSION=v1.0.0 && shx rm -rf generated/templates/mailvalidation && shx mkdir -p generated/templates/mailvalidation && curl -XGET https://raw.githubusercontent.com/pagopa/io-messages-email-templates/$VERSION/MailValidation/index.html -o $TARGET.html && sed -i --posix \"s/\\.\\.\\/assets\\//https:\\/\\/raw.githubusercontent.com\\/pagopa\\/io-messages-email-templates\\/$VERSION\\/assets\\//g\" $TARGET.html && shx echo 'export function apply(title: string, validationUrl: string): string { return `' > $TARGET.ts && shx cat $TARGET.html >> $TARGET.ts && shx echo '`;}' >> $TARGET.ts",
+    "generate:templates:login": "shx rm -rf generated/templates/login && shx mkdir -p generated/templates/login && ts-node _scripts/email_template_script.ts LoginNotification IOPID-386 \"_scripts/emailApplierTemplates/loginNotification\" \"generated/templates/login/index\"",
```

```suggestion
    "generate:templates:login": "shx rm -rf generated/templates/login && shx mkdir -p generated/templates/login && ts-node _scripts/email_template_script.ts LoginNotification IOPID-386 '_scripts/emailApplierTemplates/loginNotification' 'generated/templates/login/index'",
```
we can omit character escaping here